client/my-sites/checkout/composite-checkout/wpcom/components/wp-checkout-order-summary.js

```diff
@@ -189,7 +190,9 @@ function getPlanFeatures( plan, translate, hasDomainsInCart, hasRenewalInCart )
 		return [
 			showFreeDomainFeature && translate( 'Free domain for one year' ),
 			translate( 'Unlimited access to our library of Premium Themes' ),
-			translate( 'Subscriber-only content and simple payment buttons' ),
+			isEnabled( 'earn/pay-with-paypal' )
+				? translate( 'Subscriber-only content and Pay with PayPal buttons' )
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.